jest-setup.js

```diff
@@ -0,0 +1,4 @@
+// setup file
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-15';
+configure({ adapter: new Adapter() });
```

Comment:
If you don't mind, what does this file do? Curious to learn.

Reply:
Enzyme 3 requires this adapter to be defined. This setupFile is defined in the package.json jest object and gets run before each spec file is loaded.

Reply:
Nice, great to know. Thank you :)

Reply:
https://github.com/airbnb/enzyme/blob/master/docs/installation/react-15.md